beets/autotag/mb.py

```diff
@@ -285,6 +287,24 @@ def album_info(release):
     artist_name, artist_sort_name, artist_credit_name = \
         _flatten_artist_credit(release['artist-credit'])
 
+    ntracks = 0
+    for medium in release['medium-list']:
+        ntracks += len(medium['track-list'])
+
+    # for albums with more than 500 tracks
+    if ntracks > 500:
```

Comment:
Is there perhaps any more reliable/obvious way to detect when MusicBrainz has omitted recording information of a too-big release? For example, maybe the API will just have all the `recording` fields missing? This would be nice because it would avoid hard-coding a number that might change in the API.

Reply:
I checked but the only difference seems to be that the ```recording``` dicts contain more or less info, I tried comparing a very big release with a release that consists of one of the discs and the only meaningful differences were that the recordings of the bigger release contain no ```artist-relation-list``` and ```work-relation-list``` info, but that info cannot be used to discriminate, as it could also happen on small releases with incomplete info. 
I tried looking at the logs but there's nothing there either.

Reply:
Hmm, interesting. We may be stuck with the hard-coded threshold… could be worse, I guess. But at the very least, it would be nice to:

1. Check in with the MusicBrainz folks on IRC or similar to see if they know of any obvious way to tell when this was engaged?
2. Put the number into a module-level constant.

Reply:
I couldn't see anything, I asked on the forum and on IRC and didn't get an answer, so I guess we're stuck with 500.